textile/features.textile

```diff
@@ -116,34 +150,27 @@ h3(#restclient). RestClient
+** @(RSC15g)@ This clause has been replaced by "@RSC15n@":#RSC15n. It was valid up to and including specification version @TBD@.
+** @(RSC15h)@ This clause has been replaced by "@REC2@":#REC2. It was valid up to and including specification version @TBD@.
+** @(RSC15i)@ This clause has been replaced by "@REC2@":#REC2. It was valid up to and including specification version @TBD@.
+** @(RSC15j)@ Requests to fallback hosts must use a matching Host header as this is necessary when fallbacks are proxied through a CDN. For example, if a request to @main.realtime.ably.net@ fails and will be retried to @c.ably-realtime.com@, the Host header must be set to @c.ably-realtime.com@ in the retried request
 ** @(RSC15d)@ This clause has been replaced by "@RSC15l@":#RSC15l.
 ** @(RSC15l)@ Errors that necessitate use of an alternative host include any of the following conditions. (Resending requests that have failed for other failure conditions will not fix the problem and will simply increase the load on other datacenters unnecessarily).
 *** @(RSC15l1)@ host unresolvable or unreachable
 *** @(RSC15l2)@ request timeout
 *** @(RSC15l3)@ a response with an HTTP status code in the range @500 <= code <= 504@
 *** @(RSC15l4)@ a response with a `Server: CloudFront` header and an HTTP status code @>= 400@
-** @(RSC15f)@ Once/if a given fallback host succeeds, the client should store that successful fallback host for @ClientOptions.fallbackRetryTimeout@. Future HTTP requests during that period should use that host. If during this period a "qualifying errors":#RSC15d occurs on that host, or after @fallbackRetryTimeout@ has expired, it should be un-stored, and the fallback sequence begun again from scratch, starting with the default primary host (@rest.ably.io@ or @ClientOptions#restHost@) or, in the case of an existing fallback realtime connection as per (RTN17e), with the current fallback realtime host.
+** @(RSC15f)@ Once/if a given fallback host succeeds, the client should store that successful fallback host for @ClientOptions.fallbackRetryTimeout@. Future HTTP requests during that period should use that host. If during this period a "qualifying errors":#RSC15d occurs on that host, or after @fallbackRetryTimeout@ has expired, it should be un-stored, and the fallback sequence begun again from scratch, starting with the default primary host (@main.realtime.ably.net@ or @ClientOptions#restHost@) or, in the case of an existing fallback realtime connection as per (RTN17e), with the current fallback realtime host.
```

Comment:
> starting with the default primary host (`main.realtime.ably.net` or `ClientOptions#restHost`)

"Starting with the REC1 `Primary Domain`"

Reply:
Fixed https://github.com/ably/specification/pull/213/commits/27a16960b674d44ba91873e22f3203756e7d7900